Cargo.toml

```diff
@@ -27,10 +27,17 @@ missing_errors_doc = "allow"
 missing_panics_doc = "allow"
 too_many_lines = "allow"
 
+[profile.dev]
+split-debuginfo = "unpacked"
+
+[profile.release]
+codegen-units = 1
+lto = true
+opt-level = 2
+panic = "abort"
+strip = "debuginfo"
+
 [profile.release-tiny]
 inherits = "release"
 opt-level = "z"
-lto = true
-codegen-units = 1
-strip = "symbols"
-panic = "abort"
+strip = true
```

Comment:
`strip = true` is similar to `strip = "symbols"`. Both are doing the same thing.